thentos-adhocracy/src/Thentos/Adhocracy3/Types.hs

```diff
@@ -72,7 +72,7 @@ instance EmptyDB DB where
 
 instance DB `Core.Extends` Core.DB where
     focus f (DB db) = DB <$> f db
-    asDBThentosError = ThentosA3ErrorCore
+    thentosErrorFromParent = ThentosA3ErrorCore
```

Comment:
I tend to dislike comments and FIXMEs etc., since it usually makes it feel like it is okay to only half do a task or half implement a feature. And commented out code bitrots quickly (it is never typechecked), and soon becomes more confusing than helpful. But again, this is a very general point, and not specific to this PR.